flightlib/include/flightlib/bridges/unity_bridge.hpp

```diff
@@ -94,7 +94,7 @@ class UnityBridge {
   int64_t u_packet_latency_;
 
   // axuiliary variables
-  const Scalar unity_connection_time_out_{10.0};
+  const Scalar unity_connection_time_out_{60.0};
```

Comment:
any reason why using 60 seconds for the connection time out?

Reply:
the motion planning example times out. Loading the forest environment for the first time after launching the application can take longer than this 10 seconds. For the other environments, 10 seconds is totally sufficient.